salt/orchestrate/edx/build_ami.sls

```diff
@@ -4,6 +4,7 @@
 {% set env_settings = env_dict.environments[ENVIRONMENT] %}
 {% set VPC_NAME = salt.environ.get('VPC_NAME', env_settings.vpc_name) %}
 {% set BUSINESS_UNIT = salt.environ.get('BUSINESS_UNIT', env_settings.business_unit) %}
+{% set RUN_MIGRATIONS = salt.environ.get('RUN_MIGRATIONS') %}
```

Comment:
Should add a default value of `False` just to help clarify that it's a boolean.

Reply:
We should also remove the EDX_VERSION line above as it no longer has any purpose.

Reply:
OK, I've made those changes.